src/harberger-lib/components/Info.re

```diff
@@ -1,5 +1,342 @@
+           <br />
+         </p>;
+       | None => React.null
+       }}
+    </React.Fragment>;
+  };
+};
+module SimpleView = {
+  [@react.component]
+  let make =
+      (
+        ~monthlyRate,
+        ~tokenName,
+        ~optMonthlyPledgeEth,
+        ~unit,
+        ~currentPatron,
+        ~displayNameStr,
+        ~totalPatronage,
+        ~definiteTime,
+        ~daysHeld,
+        ~orgId,
+        ~orgName,
+        ~priceString,
+        ~optionalSpecies,
+      ) => {
+    let clearAndPush = RootProvider.useClearNonUrlStateAndPushRoute();
+
+    <React.Fragment>
+      <p>
+        {(tokenName ++ " is currently protected by ")->React.string}
+        <a
+          onClick={e => {
+            ReactEvent.Mouse.preventDefault(e);
+            clearAndPush({j|/#user/$currentPatron|j});
+          }}>
+          displayNameStr->restr
+        </a>
+        {(
+           " who values their guardianship of "
+           ++ tokenName
+           ++ " at "
+           ++ priceString
+           ++ ". "
+           ++ tokenName
+           ++ " has a monthly pledge rate of "
+           ++ monthlyRate
+           ++ "%. This means "
+         )
+         ->React.string}
+        <a
+          onClick={e => {
+            ReactEvent.Mouse.preventDefault(e);
+            clearAndPush({j|/#user/$currentPatron|j});
+          }}>
+          displayNameStr->restr
+        </a>
+        {(
+           " has to contribute "
+           ++ (
+             switch (optMonthlyPledgeEth) {
+             | Some(monthlyPledgeEth) => monthlyPledgeEth ++ " " ++ unit
+             | None => "Loading"
+             }
+           )
+           ++ " monthly to "
+         )
+         ->React.string}
+        <a
+          onClick={e => {
+            ReactEvent.Mouse.preventDefault(e);
+            clearAndPush({j|/#org/$orgId|j});
+          }}>
+          orgName->restr
+        </a>
+        {(
+           " for the protection of "
+           ++ tokenName
+           ++ optionalSpecies->Option.mapWithDefault("", species =>
+                " the " ++ species
+              )
+           ++ "."
+         )
+         ->React.string}
+      </p>
+      {switch (daysHeld) {
+       | Some((daysHeldFloat, _timeAquired)) =>
+         <p>
+           <a
+             onClick={e => {
+               ReactEvent.Mouse.preventDefault(e);
+               clearAndPush({j|/#user/$currentPatron|j});
+             }}>
+             displayNameStr->restr
+           </a>
+           {(
+              " has been the guardian of "
+              ++ tokenName
+              ++ " for "
+              ++ daysHeldFloat->Js.Float.toFixed
+              ++ " days"
+            )
+            ->React.string}
+           {switch (definiteTime) {
+            | Some(date) =>
+              <>
+                "and has enough deposit to last "->React.string
+                <CountDown endDateMoment=date />
+                {j| - remember to keep topping up that deposit 😉.|j}
```

Comment:
Have we used emoji's before? It shows as a different char in the ui for me but I feel like we did have a way to add emojis before. Not a biggy

Reply:
Firefox and chrome have different emojis (of the same emojis).

I don't think we have used emojis in wildcards before, only other places